webapp/src/actions.ts

```diff
@@ -0,0 +1,9 @@
+import * as Actions from './types/actions';
+
+export const userHasConnected = (): Actions.UserHasConnected => ({
+    type: Actions.USER_HAS_CONNECTED,
```

Comment:
nit: the websocket event is `user_connected` but here we model with `has` -- might we converge on the same phrase just so a case insensitive grep is likely to find all related matches?

Reply:
Yeah, I'll drop the `has` everywhere it's actually bothering me to have it!